easybuild/easyconfigs/f/FlexiBLAS/FlexiBLAS-3.0.4-GCC-10.3.0.eb

```diff
@@ -0,0 +1,27 @@
+name = 'FlexiBLAS'
+version = '3.0.4'
+
+homepage = 'https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release'
+description = """FlexiBLAS is a wrapper library that enables the exchange of the BLAS and LAPACK implementation
+used by a program without recompiling or relinking it."""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+local_extra_flags = "-fstack-protector-strong -fstack-clash-protection"
+toolchainopts = {'pic': True, 'extra_cflags': local_extra_flags, 'extra_fflags': local_extra_flags}
+
+source_urls = ['https://csc.mpi-magdeburg.mpg.de/mpcsc/software/flexiblas/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['50a88f2e88994dda91b2a2621850afd9654b3b84820e737e335687a46751be5c']
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('Python', '3.9.5'),  # required for running the tests
+]
+
+# note: first listed library will be used as default by FlexiBLAS,
+# unless otherwise specified via -DFLEXIBLAS_DEFAULT in configopts
```

Comment:
```suggestion
# unless otherwise specified via easyconfig parameter flexiblas_default
```

Reply:
done in https://github.com/easybuilders/easybuild-easyconfigs/pull/12476/commits/5daaacbd965e207556362015d376efa3e314bc98